daal4py/sklearn/metrics/_ranking.py

```diff
@@ -158,6 +158,7 @@ def _daal_roc_auc_score(
         if _dal_ready:
             if not np.array_equal(labels, [0, 1]) or labels.dtype == bool:
                 y_true = label_binarize(y_true, classes=labels)[:, 0]
+                y_score = label_binarize(y_score, classes=labels)[:, 0]
```

Comment:
There is a fix for test fails

```suggestion
                if hasattr(y_score, 'dtype') and y_score.dtype == bool:
                    y_score = label_binarize(y_score, classes=labels)[:, 0]
```